packages/controllers/src/snaps/SnapController.test.ts

```diff
@@ -359,28 +358,24 @@ const getSnapManifest = ({
 };
 
 const getSnapObject = ({
-  blocked = false,
-  enabled = true,
   id = MOCK_SNAP_ID,
   initialPermissions = {},
   manifest = getSnapManifest(),
   permissionName = `wallet_snap_${MOCK_SNAP_ID}`,
   sourceCode = MOCK_SNAP_SOURCE_CODE,
-  status = SnapStatus.stopped,
+  status = 'stopped' as Status,
```

Comment:
I think we should continue using an `enum` for the status. Any reason in particular that you switched?

Reply:
As it turns out, `xstate` won't accept an `enum`. The status is typed via a string literal union, so this ought to be fine so long as the statuses aren't used externally. If so, we should use a manual "enum object" for easier use in JavaScript contexts.

Reply:
On second thought, we probably will want to consume the types externally, specifically in the UI, where we will look at the snap's status to decide how to display certain components. Therefore, an enum object would be preferable.